cpu/sam0_common/include/periph_cpu_common.h

```diff
@@ -945,6 +945,18 @@ typedef struct {
  */
 #define WDT_HAS_INIT                   (1)
 
+/**
+ * @brief   Frequency meter configuration
+ *
+ */
+#ifdef MODULE_PERIPH_FREQM
+struct freqm_config {
+    gpio_t msr_clk_src;     /*< GPIO at which the frequency is to be measured */
+    uint8_t ref_clk_src;    /*< GCLK source select for reference */
```

Comment:
```suggestion
    uint8_t gclk_src;       /*< GCLK source select for reference */
```

Other peripherals already use the `.gclk_src` for this.